packages/nuxt/src/app/composables/lazy-hydration.ts

```
@@ -0,0 +1,14 @@
+import type { AsyncComponentLoader, Component, ComponentPublicInstance, DefineComponent } from 'vue'
+
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+type LazyHydrationComponent<T extends Component, Props> = T & DefineComponent<Props, {}, {}, {}, {}, {}, {}, { hydrated: () => void }>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)